docs/plugins/api.rst

```diff
@@ -1,7 +1,20 @@
 Plugin API
 ==========
 
-Plugins can affect the behaviour of Tutor at multiple levels. First, plugins can define new services with their Docker images, settings and the right initialisation commands. To do so you will have to define custom :ref:`config <plugin_config>`, :ref:`patches <plugin_patches>`, :ref:`hooks <plugin_hooks>` and :ref:`templates <plugin_templates>`. Then, plugins can also extend the CLI by defining their own :ref:`commands <plugin_command>`.
+A Tutor plugin is defined either by a YAML file or a Python package.
```

Comment:
I feel like this sentence should be placed after the more general description of what plugins can achieve. 

I propose the following:

```
Plugins can affect the behaviour of Tutor at multiple levels:

* Add new settings or modify existing one in the Tutor configuration (see :ref:`config <plugin_config>`).
* Add new templates to the Tutor project environment or modify existing ones (see :ref:`patches <plugin_patches>`, :ref:`templates <plugin_templates>` and :ref:`hooks <plugin_hooks>`).
* Add custom commands to the Tutor CLI (see :ref:`command <plugin_command>`).

There exists two different APIs to create Tutor plugins: either with YAML files or Python packages. YAML files are more simple to create, but are limited to just configuration and template patches.
```

Note that you should not to line-wrap the docs. (and below as well)

Reply:
That it much more streamlined, thanks.